### ISSUE TITLE
Update Steam API Key

### ISSUE THREAD

**Issue by kevinvanrijn** [Foohy/wheelofsteamgames#1]
Looks like I'm late to the party.

Did the Steam API key expire? Or is something else going on?

**Comment by Invalid-Vertex** [comment]
Seems the API key did indeed expire, as after compiling with my API key, it works fine.

**Comment by Foohy** [comment]
Yep, sorry. I had to revoke my API key for another project as it had to be private. I might be able to update it with a key of another account though.

**Comment by KIPdeKIP** [comment]
Feel free to use my alt's key. I'm not using it for anything.

![Key](https://cloud.githubusercontent.com/assets/6368561/13149363/964b211a-d661-11e5-9dc1-ce28c94d50a1.png)

**Comment by maimihn** [comment]
Please update it, I know you have a lot of things going like tower unite but this is the only astheticly pleasing steam wheel out there and the only .exe.

**Comment by dominateeye** [comment]
Yeah, I'd really like to use this too because PixelTail is rad, but it's returning the 403 error every time.

**Comment by NekoAlosama** [comment]
[The issuer has made a fork](https://github.com/EIGHTFINITE-Software/wheel-of-steam-games/releases/tag/1.1.0) with added missing libraries, a release compiled with a different API key, and instructions in the README.md on where to change the API key when building.
